source/panel_predictor/__main__.py

```diff
@@ -13,12 +13,11 @@ def display_frame(frame, target=None):
     cv2.imshow('Press q to quit', frame)
 
 
-def run_video(video_path, framework):
+def run_video(framework, capture):
```

Comment:
It makes more sense to pass a `PanelPredictor` object instead of `framework`. That way, the `PanelPredictor` object can be initialized to your liking before being passed.